packages/api/src/command/medical/patient/consolidated-get.ts

```diff
@@ -110,7 +130,68 @@ export async function startConsolidatedQuery({
   return progress;
 }
 
-async function getConsolidatedAndSendToCx(params: GetConsolidatedParams): Promise<void> {
+function appendProgressToProcessingQueries(
+  currentConsolidatedQueries: ConsolidatedQuery[] | undefined,
+  progress: ConsolidatedQuery
+): ConsolidatedQuery[] {
+  if (currentConsolidatedQueries) {
+    const queriesInProgress = currentConsolidatedQueries.filter(
+      query => query.status === "processing"
+    );
+
+    return [...queriesInProgress, progress];
+  }
+
+  return [progress];
+}
+
+export function getCurrentConsolidatedProgress(
```

Comment:
I think we either rename this function to indicate we're only returning the current `processing`  progress, or remove the filter for `processing` and leave this to the client's to decide what to do with it.

I suggest the latter, so this function can be reused in other situations as well.

Reply:
Added param with `progressStatus = "processing"` as default